libs/ui-lib/lib/ocm/components/clusterConfiguration/OpenShiftVersionModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, ButtonVariant, Modal, ModalVariant } from '@patternfly/react-core';
+import './OpenshiftVersionModal.css';
+import { OpenshiftSelectWithSearch } from '../../../common/components/ui/OpenshiftVersionSelect';
+import { useOpenshiftVersions } from '../../hooks';
+import { OpenshiftVersionOptionType } from '../../../common';
+
+type OpenShiftVersionModalProps = {
+  setOpenhiftVersionModalOpen: (isOpen: boolean) => void;
+  isOpen: boolean;
+  setValueSelected: (valueSelected: OpenshiftVersionOptionType) => void;
+};
+
+export const OpenShiftVersionModal = ({
+  setOpenhiftVersionModalOpen,
+  isOpen,
+  setValueSelected,
+}: OpenShiftVersionModalProps) => {
+  const { versions } = useOpenshiftVersions('false');
```

Comment:
Since the `OpenShiftVersionModal` is being rendered unconditionally, we'd always been fetching all versions, even if most users may select among latest versions.

I think it would be best to only render `OpenShiftVersionModal` when users want to see those additional versions.